### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by hntrhfr**

### CODE REVIEW

File: services/api/src/main.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { AppModule } from "./app.module";
 
 export function configureApp(app: INestApplication): void {
   app.use(cookieParser());
-  app.useGlobalPipes(new ValidationPipe());
+  app.useGlobalPipes(new ValidationPipe({ forbidUnknownValues: false }));
```
Comment (thomasdax98) [inline services/api/src/main.ts:13]: Why did you add `forbidUnknownValues: false`? According to my knowledge, that should protect against attacks. So if we have a problem there, we should probably fix the root cause

**Review by thomasdax98 — CHANGES_REQUESTED** [review]

File: services/api/src/main.ts (hunk 1 above)
Comment (lucapircher) [reply services/api/src/main.ts:13]: There was a breaking change in the default configuration of the package. Prior to the update the option was off by default.

https://github.com/typestack/class-validator/blob/develop/CHANGELOG.md#breaking-changes

So we are currently just restoring that behaviour. I agree that we should investigate why this valdiation fails and resolve the issues. But that is beyond the scope of this task.

**Review by thomasdax98 — APPROVED** [review]
